specification/under_development/output_schemas/No_Labels_And_Ordered.md

```diff
@@ -0,0 +1,266 @@
+# Ordered Output Format with Optional Labels
+
+For systems that don't support string parameters, the ordered output format can be used. This format omits any labels that are passed through output recording functions. A [grammar](./Grammars.md#no-labels-and-ordered) for this format is available which defines the structure and valid values.
+
+The format specifies what the [output recording](../profiles/Base_Profile.md#output-recording) functions emit based on the selected schema and entry point [attributes][].
+
+## Record types
+
+Records are expected to have 1-3 fields:
+
+- RecordType: an id that identifies the type of content in the record. The only supported RecordTypes will be `START`, `METADATA`, `OUTPUT`, and `END`.
+  - `START` indicates the beginning of a shot's output definition
+  - `METADATA` records have a name field and optional value field
+  - `OUTPUT` records have a type in the set: `RESULT`, `TUPLE`, `ARRAY`, `INT`, `BOOL`, or `DOUBLE`.
+    - `TUPLE` and `ARRAY` types have a third field indicating the number of items in the container.
+    - `RESULT`, `INT`, `BOOL`, `DOUBLE` types have a third field indicating the value of the record.
+  - `END` indicate the completion of a shot's output definition with the exit code for the entry point.
+
+Output for each shot is expected to have:
+
+- One `START` record
+- `METADATA` records for each of the defined entry point attributes. The generation of these records is not part of the generated QIR, but written as part of the [base profile][] spec as pass-thru metadata. The [base profile required attributes](../profiles/Base_Profile.md#attributes) define the minimum set of attributes which will appear. Examples can be found in the [notes for implementors examples](./Notes_For_Implementors.md#examples).
+- One or more `OUTPUT` records
+- One `END` record
+
+The `START` and `END` records define a shot, but are not part of the generated QIR. They are produced as part of the provider's recording of a shot's output. The `START` record does not have any value entry. The `END` record has a value entry of `0`. The `OUTPUT` records will have a value corresponding to the runtime function called, either a result type record or a string representation of a supported primitive type. The `OUTPUT` records must appear in the same order as the calls in the QIR program.
+
+Example log for a single shot:
+
+```log
+START
+METADATA\tuser_metadata1_name_only
+METADATA\tuser_metadata2_name\tuser_metadata2_value
+METADATA\tuser_metadata3_name\tuser_metadata3_value
+METADATA\tentry_point
+METADATA\tnum_required_qubits\t5
+METADATA\tnum_required_results\t5
```

Comment:
To be consistent with the Base Profile documentation, I believe this should be `required_num_qubits` instead of `num_required_qubits` and `required_num_results` instead of `num_required_results`.